wire/protocol.go

```diff
@@ -183,6 +183,9 @@ const (
 	// TestNet3 represents the test network (version 3).
 	TestNet3 BitcoinNet = 0x0709110b
 
+	// SigNet represents the signet (default).
```

Comment:
I know this is probably the way it's called in most places (`signet (default)`), though I think we could make it a bit more obvious that this is only one of many possible signets, we could add something like: `// SigNet represents the public default SigNet. For custom signets, see `CustomSignetParams`.

Reply:
Thanks! Fixed.